petastorm/etl/legacy.py

```diff
@@ -43,5 +69,5 @@ def depickle_legacy_package_name_compatible(pickled_string):
                                'Regenerate metadata.', legacy_package_name, legacy_module, legacy_module)
 
             pickled_string = modified_pickled_string
-
+    restricted_loads(pickled_string)
```

Comment:
We are ignoring the return value here. Also, the implementation fails the tests. Started https://github.com/uber/petastorm/pull/640 to resolve test issues.